sources/packages/web/src/views/supporting-user/SupportingInformation.vue

```diff
@@ -158,10 +158,32 @@ export default defineComponent({
       // Disable internal submit button.
       formioUtils.disableWizardButtons(formInstance);
       formInstance.options.buttonSettings.showSubmit = false;
+      // Handle the navigation using the breadcrumbs.
+      formInstance.on(
+        "wizardPageSelected",
+        (page: WizardNavigationEvent, index: number) => {
+          isFirstPage.value = index === 0;
+          isLastPage.value = formInstance.isLastPage();
+          // Event to set isInFirstPage, current page and isInLastPage to parent.
+          context.emit(
```

Comment:
Are these ones (lines 168 and 181) still to be removed?

Reply:
Yeah, you're right. The emits are not necessary here, only set the control variables. Thanks!